fastlane/Fastfile

```diff
@@ -362,9 +361,7 @@ platform :android do
     version = release_version_current
 
     # Wrap up
-    remove_branch_protection(repository: GITHUB_REPO, branch: "release/#{version}")
```

Comment:
This is already done later during `publish_release`, which is a better moment to remove the release branch protection anyway (see https://github.com/Automattic/apps-infra-plans/issues/83)